c2corg_ui/static/js/imageuploader.js

```diff
@@ -306,7 +306,7 @@ app.ImageUploaderController.prototype.save = function() {
       scope['photo'] = images[i];
       scope['photo']['image_id'] = id;
       scope['photo']['edit_url'] = '/images/edit/' + imageIds[i]['document_id'] + '/' + images[i]['locales'][0]['lang'];
-      scope['photo']['view_url'] = this.url_.buildDocumentUrl('images', imageIds[i]['document_id'], images[i]['locales'][0], images[i]['locales'][0]['lang'] );
+      scope['photo']['view_url'] = this.url_.buildDocumentUrl('images', imageIds[i]['document_id'], images[i]['locales'][0], images[i]['locales'][0]['lang']);
```

Comment:
This change has already been done by @tsauerwein in a previous PR. See https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/imageuploader.js#L309

It's weird that github doesn't report a conflict!? Have you rebased your branch?